### PR TITLE
Newsweek development branch merge

### PR DESCRIPTION
**Pull request by eliasecasta**
The goal of this project was to learn about Bootstrap and to apply it to create 3 different web views to clone the Newsweek site.

IMPORTANT NOTE: PLEASE CLONE THE PROJECT, I AM NOT SEEING SOME OF THE PROBLEMS THAT YOU GUYS ARE SEEING, GITHACK IS DISPLAYING DIFFERENTLY AS HOW I BUILT IT FOR SOME REASON.

NOTE 2: THE WEB PAGE WAS DESIGNED TO WORK WITH 4 BREAKPOINTS: BOOTSTRAP: SM/MD/LG AND WITHOUT THE INSPECT WINDOW. I DIDNT ADAPT IT TO CUSTOM WIDTHS.

### CODE REVIEW

**Comment by voscarmv** [comment]
# Pull Request Under Review by Oscar Mier (TSE) 👨‍💻 🐦

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, @e71az ! 🖥️ 👀 ✨ 
## However, some changes are still required... 🧰 🔨 🔧 
## On your `README.md` file
- The screenshot images are too long! You don't need to have the **whole** page on the `README.md`. It's enough to have a small sample.
## Page layout
- The navbar and content of the original page are wider, and have a different background color on the sides, than your version. Please make your page more similar to the original by changing these details.
####  👇 Your version 👇
image
####  👇 Original page 👇 
image

## Below the Top Story section
- Make sure the articles are aligned with the section titles (HEROES OF THE PANDEMIC, etc) so that they look the same as on the original page. Also, add the bottom-left corner tags to the article images.
####  👇 Your version 👇
image
####  👇 Original page 👇
image
- Use the [Sticky Top](https://getbootstrap.com/docs/4.0/utilities/position/#sticky-top) Bootstrap effect on the Featured Story, Top Story and The Debate sections, to get that sticky scrolling effect we see on the original Newsweek page.
## Subscribe and Newsletter sections
- Your version is missing the Subscribe and Newsletter sections near the footer of the page. Please add them.
####  👇 Your version 👇
image
####  👇 Original page 👇 
image

### Submit these changes and request a new code review! 😄 👍 
### Questions of comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1687 @@
...
+        </li>
+      </ul>
+      <ul class="nav navbar-nav ml-auto">
+        <li class="nav-item">
+          <a class="nav-link" href="#">Login</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav> -->
```
Comment (BrittanyBlake) [inline index.html]: Please remove commented code

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @e71az! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****
### Navigation
- [x] The navigation links should span the entire width of the page. As requested by the previous TSE, please compare it with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-09 at 9 08 09 PM
The original page 👇 
Screen Shot 2020-06-09 at 9 08 16 PM

- [x] Your subscribe button appears to be a screenshot. Please build the subscribe button using HTML, CSS, and Bootstrap classes.
Screen Shot 2020-06-09 at 9 17 08 PM
### Below top story

- [x] As requested by the previous TSE, please add the tags on the bottom-left corner of the images.
The original page 👇 
Screen Shot 2020-06-09 at 9 07 19 PM
Your clone 👇 
Screen Shot 2020-06-09 at 9 07 08 PM

### Subscribe and Newsletter

- [x] The font-color for the subscribe and newsletter headings on your clone is not the same as on the original page. Please fix this using bootstrap classes.
The original page 👇 
Screen Shot 2020-06-09 at 9 06 41 PM
Your clone 👇 
Screen Shot 2020-06-09 at 9 06 50 PM

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @e71az   :wave:
-
You have done a splendid job with the changes! :tada: Keep it up.
We still do have some few changes to carry out before we proceed (see review comment).
##
### [The clone](https://raw.githack.com/e71az/newsweek/development/index.html) shows a good understanding and use of bootstrap 💯 

###  Kindly take your time and see the difference below 👇 then work on the changes as seen.
## Fix the navbar

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/e71az/newsweek/development/index.html)|
|:---:|:---:|
|source|clone|

- [x] Kindly ensure the main navigation links ([link to page](https://www.newsweek.com/)) are as well spaced out as with the source page.

## Fix the `in the magazine` section
|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/e71az/newsweek/development/index.html)|
|:---:|:---:|
|in-the-magazine|in-the-magazine-clone|

- [x] There is currently too much space in the `in the magazine` section. Kindly let's get rid of them.

## Fix the Footer section. 

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/e71az/newsweek/development/index.html)|
|:---:|:---:|
|footer|footer-clone|

- [x] The Subscribe and newsletter sections' buttons are not properly styled. The header colours should be red in colour as well. While at it lets also increase the spacing in the icons.

- [x] We still have some commented out code in your markup. It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
 
## OPTIONAL CHANGES
|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/e71az/newsweek/development/index.html)|
|:---:|:---:|
|source|clone|

- [x] The general layout is okay. We can always improve on it further. I'd suggest the use of the container class to centre the whole page appropriately to match the source page.

- [ ] Consider reducing the height of the navbar to match that of the course page.

- [x] Validate your CSS file using this[ tool](https://jigsaw.w3.org/css-validator/validator).

##
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @e71az ! 👋 

Great job so far!👍 You have also done a wonderful job implementing the changes requested by myself and the other TSEs 🎖️ You are almost done! 💪 🏗️  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] You have done a wonderful job fixing the width of the nav links 👍  however, the spacing for a few of the elements still needs to be improved. Please compare your clone with the original and fix this.
The original page 👇 
Screen Shot 2020-06-11 at 2 58 38 AM
Your clone 👇 
Screen Shot 2020-06-11 at 2 58 30 AM

- [x] Please ensure the subscribe button on the small screen matches that of the original page. 
Your clone 👇 
Screen Shot 2020-06-11 at 3 03 33 AM
The original page 👇 
Screen Shot 2020-06-11 at 3 03 49 AM


- [x] There is still too much space in your "in the magazine section" as was highlighted by a previous TSE. Please compare your page with the original and fix this issue. 
Your clone 👇 
Screen Shot 2020-06-11 at 3 00 14 AM
The original page 👇 
Screen Shot 2020-06-11 at 3 00 26 AM

- _hint: the bottom of the elements should be aligned and the images should be taller_



****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Impressive project @e71az 👏🙌

### However, we still need to do some changes before we can move ahead.

- In the `second-nav` section, some elements are breaking lines.
  
  Annotation 2020-06-11 223409

 - Bottom-left corner tag is still missing, as pointed out by the previous reviewer.
  
   Right now 👇
   Annotation 2020-06-11 223545

   Original 👇
   
   Annotation 2020-06-11 223736

- In `In the magazine` section, the cards should be of the same height as the magazine cover. Increase their height as well.
 Right now 👇

 Annotation 2020-06-11 223857

  Original 👇
  Annotation 2020-06-11 223923

- Bottom-left tag is still missing in the `Editor's picks` section
   Right now 👇

   Annotation 2020-06-11 224243

  Original 👇

Annotation 2020-06-11 224222

- Add space above `heroes-pandemic` section
 
Annotation 2020-06-11 224523

-  `heroes-pandemic` section needs to be full width.

    Right now 👇

    Annotation 2020-06-11 224652

    Original 👇

    Annotation 2020-06-11 224911


- Decrease font size in `the-debate` section.
  
   Annotation 2020-06-11 225114

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1729 @@
...
+            class="col d-flex right-nav align-items-center justify-content-end"
+          >
+            <i class="fa fa-user mx-1 d-md-none"></i>
+            <h5 class="d-none d-md-block nav-login mt-2">
+              <a href="#"> LOGIN</a>
+            </h5>
+            <button type="submit" class="btn btn-warning mx-3 py-1 px-2">
+              SUBSCRIBE >
+            </button>
+            <!-- <img class="mx-1 mt-1" src="images/subscribe.png" alt="subscribe" /> -->
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove commented code. It’s better to delete commented code. If you need to review a previous version of your code, use git history (screenshot from Pathwright)
  commented code

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply these changes and submit another code review.

Happy coding 👨‍💻

File: index.html (hunk 2 above)
Comment (eliasecasta) [reply index.html]: Hello Ghosh, can you open slack and check my message please? Thank you!

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required  🏗️ 🔨 
****
### Hello @e71az  👋 
You are doing a great job with this project! 👍 You are almost done! 💪 Great job applying the feedback you have received from myself and the other TSEs. 🎖️ 🌟 There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Your navigation links extend beyond the width of the page. Please take a look at the screenshot below paying close attention to the search bar and fix this. 👇
Screen Shot 2020-06-11 at 9 07 37 PM
Screen Shot 2020-06-11 at 9 17 46 PM

- [x] Your magazine cover in the magazine section is missing the date. Please take a look at the screenshots below and fix this
Your clone 👇 
Screen Shot 2020-06-11 at 9 08 20 PM
The original page 👇 
Screen Shot 2020-06-11 at 9 08 29 PM
****

- Please note that the live demo link in your README is not updated. Please try to update it as it is still showing all of the previous errors. In order to properly do this review, I needed to download your project and open a live server. 

****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi @e71az ! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**